delta/plugins/elasticsearch/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/model/ElasticSearchViewSearchParams.scala

```diff
@@ -0,0 +1,37 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model
+
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model
+import ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch.model.ElasticSearchViewType._
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import ch.epfl.bluebrain.nexus.delta.sdk.model.ResourceRef
+import ch.epfl.bluebrain.nexus.delta.sdk.model.identities.Identity.Subject
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+import ch.epfl.bluebrain.nexus.delta.sdk.model.search.SearchParams
+
+/**
+  * Search parameters for ElasticSearch views.
+  *
+  * @param project    the optional parent project of the views
+  * @param deprecated the optional deprecation status of the views
+  * @param rev        the optional revision of the views
+  * @param createdBy  the optional subject who created the views
+  * @param updatedBy  the optional subject who last updated the views
+  * @param types      the collection of types to consider, where empty implies all views
+  * @param filter     an additional resource filter
+  */
+final case class ElasticSearchViewSearchParams(
+    project: Option[ProjectRef] = None,
+    deprecated: Option[Boolean] = None,
+    rev: Option[Long] = None,
+    createdBy: Option[Subject] = None,
+    updatedBy: Option[Subject] = None,
+    types: Set[Iri] = Set(ElasticSearch, AggregateElasticSearch).map(_.iri),
```

Comment:
The way the types are matches won't work with what you have here. Either change the set here or change the `SearchParams.matches(resource)`

From:
```scala
types.subsetOf(resource.types)
```

To:
```scala
resource.types.subsetOf(types)
```